test/verify/check-connection

```diff
@@ -407,21 +407,31 @@ G_MESSAGES_DEBUG=all XDG_CONFIG_DIRS=/usr/local %s -p 9999 -a 127.0.0.90 --local
     @skipImage("Kernel does not allow user namespaces", "centos-7", "rhel-7-6", "rhel-7-7", "debian-stable", "debian-testing")
     @skipImage("Added in 184", "rhel-7-6-distropkg")
```

Comment:
This now needs to be bumped; it's best to replace the version with a PR number, so that it's easier to track also across backports/stable branches. I. e. "Simple paths added in PR #11701".